### ISSUE TITLE
Predicted noise has a systematically low magnitude

### ISSUE THREAD

**Issue by matschreiner** [albarji/toy-diffusion#1]
Hi! Great code, thanks for the implementation.
A bit of a theory question, but I tried to run your code and was printing the std() of the noise and predicted_noise during the training step. 
The predicted noise seems to converge around 0.7 whereas the real noise is always around 1. I guess that since it is basically impossible to guess the correct noise the model can minimise the loss by predicting a shorter noise vector, but it's not a perfect model then. Do you know what is going on and how to fix the 'problem'?
